inst/rmarkdown/templates/meningitis_outbreak/skeleton/skeleton.Rmd

```diff
@@ -21,8 +21,41 @@ report for meningitis.
     deleting bits where variables are created/manipulated, or at least update
     them appropriatley
 - For a more detailed exaplanation of this template, see [Wiki](https://github.com/R4EPI/sitrep/wiki)
-- ADD LIST ON WHICH PACKAGES WILL BE USED FOR WHAT REASONS! AND HOW TO INSTALL
-- OTHER INFO TO ADD??
+
+
+## Installing and loading required packages 
+
+Several packages are required for different aspects of  analysis with *R*. 
+You will need to install these before starting. 
+We will be using the following packages. Some of these packages automatically
+install other packages they need to work (called dependencies).
+
+- *knitr*: for creating output documents
+- *dplyr*: for cleaning/shaping data
+- *forcats*: for cleaning/shaping data
+- *ggplot2*: for plotting charts
+- *sitrep*: for MSF field epi functions
+- *incidence*: for epicurves
+- *aweek*: for creating epiweeks
+- *epitools*: for 2x2 tables and other epi goodness
+- *sf*: for encoding spatial vector data
+- *ggspatial*: for plotting maps
+
+
+
+```{r, echo = T, eval=FALSE, results='hide', message=FALSE, warning=FALSE}
+# Installing required packages for this case study
+required_packages <- c("knitr", "dplyr", "forcats", "ggplot2",
```

Comment:
I would have these packages one per line and have the # "reason for using"  next to them, because ppl will probably delete the intro text above!